src/Codeception/Lib/Connector/Guzzle.php

```diff
@@ -224,25 +216,53 @@ protected function extractHeaders(BrowserKitRequest $request)
         return $headers;
     }
 
-    protected function extractBody(BrowserKitRequest $request)
+    protected function extractFormData(BrowserKitRequest $request)
     {
-        if (in_array(strtoupper($request->getMethod()), array('GET','HEAD'))) {
+        if (!in_array(strtoupper($request->getMethod()), ['POST', 'PUT', 'PATCH'])) {
             return null;
         }
+
+        // guessing if it is a form data
+        $headers = $request->getServer();
+        if (isset($headers['HTTP_CONTENT_TYPE'])) {
```

Comment:
It seems that the content type can be passed as $server['CONTENT_TYPE'].
extractHeaders() expects to get it in that format, extractFormData should do the same thing, right?